packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -224,14 +224,49 @@ on the `.Tooltip__arrow` to control it and prevent conflicts with the default CS
+
+```html
+<div class="spirit-feature-tooltip-enable-data-placement">
+  <div class="TooltipWrapper d-inline-block" data-spirit-element="tooltip-wrapper">
+    <button type="button" id="tooltip-trigger" data-spirit-toggle="tooltip" data-spirit-target="#floating-ui-example">
+      Toggle tooltip
+    </button>
+    <div
+      id="floating-ui-example"
+      class="Tooltip is-hidden"
+      data-spirit-placement="bottom"
+      data-spirit-flip="true"
+      data-spirit-flip-cross-axis="true"
+      data-spirit-flip-fallback-placements="bottom, right, top"
+      data-spirit-placement-controlled
+    >
+      Hello there!
+      <span class="Tooltip__arrow" data-spirit-element="arrow"></span>
+    </div>
+  </div>
+</div>
+```
 
 👉 Please consult [Floating UI][floating-ui] documentation to understand how it
 works and to get an idea of all possible cases you may need to cover.
 
+## Floating UI Attributes
+
+| Attribute                                       | Type                                         | Default  | Required | Description                                                                                                                                                                                                                                                             |
+| ----------------------------------------------- | -------------------------------------------- | -------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data-spirit-flip-cross-axis`                   | ["true" \| "false"]                          | "false"  | ✕        | This is the [cross axis][floating-ui-flip-crossaxis] in which overflow is checked to perform a flip, the axis perpendicular to main axis                                                                                                                                |
+| `data-spirit-flip-fallback-axis-sids-direction` | ["none" \| "start" \| "end" ]                | "none"   | ✕        | Whether to allow [fallback to the opposite axis][floating-ui-flip-fallbackaxissidedirection] if no placements along the preferred placement axis fit, and if so, which side direction along that axis to choose. If necessary, it will fallback to the other direction. |
+| `data-spirit-flip-fallback-placements`          | string                                       | -        | ✕        | This describes an [explicit placements][floating-ui-flip-fallbackplacements] to try if the initial placement doesn’t fit on the axes in which overflow is checked. For example you can set `"top, right, bottom"`                                                       |
```

Comment:
```suggestion
| `data-spirit-flip-fallback-placements`          | string                                       | -        | ✕        | This describes a list of [explicit placements][floating-ui-flip-fallbackplacements] to try if the initial placement doesn’t fit on the axes in which overflow is checked. For example you can set `"top, right, bottom"`                                                       |
```

? Because there is `an` with plural…

Reply:
fixed